docs/tutorials/tutorial1.md

```diff
@@ -272,33 +274,30 @@ janis translate tools/alignment.py wdl
 We'll run the workflow against the current directory.
 
 ```bash
-janis run -o . --engine cwltool \
+janis run -o tutorial1 --engine cwltool \
```

Comment:
I think the intention is to keep tutorial 1 files within the `tutorial1` directory. The alternative command in this tutorial creates `tutorial1-run-with-cromwell`, so after this change we will have both `tutorial1-run-with-cromwell` and `tutorial1`.